src/framework/loader.ts

```diff
@@ -20,13 +24,15 @@ export interface ITestNode {
 
 export interface IEntry {
   suite: string;
+  // What is this path? a path to a test module or a path to a test in that module?
   path: string;
 }
 
 interface IPendingEntry extends IEntry {
   node: Promise<ITestNode>;
 }
 
+// This is a generator of IPendingEntries? Or a generator of IterableIterator<IPendingEntry>?
```

Comment:
Generator of IPendingEntry. Just like an async function has to have return type `Promise<...>`, generators have to have an `Iterator<...>` return type.

Reply:
Ack